wemake_python_styleguide/logic/naming/enums.py

```diff
@@ -32,9 +50,13 @@ def _has_one_of_base_classes(
     return any(enum_base in string_bases for enum_base in base_names)
 
 
-def has_enum_base(defn: ast.ClassDef) -> bool:
-    """Tells whether some class has `Enum` or similar class as its base."""
-    return _has_one_of_base_classes(defn, _ENUM_NAMES)
+def has_regular_enum_base(defn: ast.ClassDef) -> bool:
+    """Tells whether some class has `Enum` or similar class as its base.
+
+    Unlike ``has_enum_base`` it excluded `IntEnum`, `StrEnum`, `IntFlag`.
```

Comment:
please, fix the doc here and in one more place: `has_enum_base` does not exist anymore.